### PR TITLE
lxc-network, lxc-profile: add page

### PR DESCRIPTION
**Pull request by mtausig**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

4.0.7

### CODE REVIEW

File: pages/linux/lxc-network.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# lxc network
+
+> Manage networks for LXD containers.
+> More information: <https://linuxcontainers.org/lxd/docs/master/networks>.
+
+- List available networks:
+
+`lxc network list`
+
+- Show configuration of an existing network:
```
Comment (navarroaxel) [inline pages/linux/lxc-network.md:10]:
```suggestion
- Show configuration of a specific network:
```

File: pages/linux/lxc-network.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`lxc network show {{network_name}}`
+
+- Add a running instance to a network:
```
Comment (navarroaxel) [inline pages/linux/lxc-network.md:14]: Should be running?
```suggestion
- Add a running instance to a specific network:
```

File: pages/linux/lxc-network.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`lxc network attach {{network_name}} {{container_name}}`
+
+- Create a new managed network:
+
+`lxc network create {{network_name}}`
+
+- Set bridge interface of a network:
```
Comment (navarroaxel) [inline pages/linux/lxc-network.md:22]:
```suggestion
- Set a bridge interface of a specific network:
```

File: pages/linux/lxc-network.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`lxc network set {{network_name}} bridge.external_interfaces {{eth0}}`
+
+- Disable NAT for a network:
```
Comment (navarroaxel) [inline pages/linux/lxc-network.md:26]:
```suggestion
- Disable NAT for a specific network:
```

File: pages/linux/lxc-profile.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# lxc profile
+
+> Manage profile for LXD containers.
+> More information: <https://linuxcontainers.org/lxd/docs/master/profiles>.
+
+- List available profiles:
+
+`lxc profile list`
+
+- Show configuration of a profile:
```
Comment (navarroaxel) [inline pages/linux/lxc-profile.md:10]:
```suggestion
- Show configuration of a specific profile:
```

File: pages/linux/lxc-profile.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`lxc profile show {{profile_name}}`
+
+- Edit profile with editor:
```
Comment (navarroaxel) [inline pages/linux/lxc-profile.md:14]:
```suggestion
- Edit a specific profile in the default editor:
```

File: pages/linux/lxc-profile.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`lxc profile edit {{profile_name}}`
+
+- Edit profile by importing a file:
```
Comment (navarroaxel) [inline pages/linux/lxc-profile.md:18]:
```suggestion
- Edit a specific profile importing the configuration values from a file:
```

File: pages/linux/lxc-profile.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`lxc profile edit {{profile_name}} < {{config.yaml}}`
+
+- Launch a new container with profiles:
```
Comment (navarroaxel) [inline pages/linux/lxc-profile.md:22]:
```suggestion
- Launch a new container with specific profiles:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]
Remember that you can apply all the suggestions using **1 commit** with `add to batch` <https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request>.
image

**Comment by mtausig** [comment]
> Remember that you can apply all the suggestions using **1 commit** with `add to batch` 

Thanks, I overlooked that.
But nevermind, I'll just squash them when the reviews are done.

**Comment by navarroaxel** [comment]
@mtausig Don't worry, we squash when we merge.

File: pages/linux/lxc-network.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# lxc network
+
+> Manage networks for LXD containers.
+> More information: <https://linuxcontainers.org/lxd/docs/master/networks>.
+
+- List available networks:
+
+`lxc network list`
+
+- Show configuration of a specific network:
```
Comment (marchersimon) [inline pages/linux/lxc-network.md:10]:
```suggestion
- Show the configuration of a specific network:
```

File: pages/linux/lxc-profile.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
+# lxc profile
+
+> Manage profile for LXD containers.
+> More information: <https://linuxcontainers.org/lxd/docs/master/profiles>.
+
+- List available profiles:
```
Comment (marchersimon) [inline pages/linux/lxc-profile.md:6]:
```suggestion
- List all available profiles:
```

File: pages/linux/lxc-profile.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
+# lxc profile
+
+> Manage profile for LXD containers.
```
Comment (marchersimon) [inline pages/linux/lxc-profile.md:3]:
```suggestion
> Manage profiles for LXD containers.
```

File: pages/linux/lxc-profile.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://linuxcontainers.org/lxd/docs/master/profiles>.
+
+- List available profiles:
+
+`lxc profile list`
+
+- Show configuration of a specific profile:
```
Comment (marchersimon) [inline pages/linux/lxc-profile.md:10]:
```suggestion
- Show the configuration of a specific profile:
```

**Review by marchersimon — APPROVED** [review]
Thanks @mtausig
There are just a few minor things I'll quickly apply myself

**Pull request merged**